components/team_controller/index.ts

```diff
@@ -57,25 +58,25 @@ function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
     };
 }
 
-function mapDispatchToProps(dispatch: Dispatch) {
-    return {
-        actions: bindActionCreators<ActionCreatorsMapObject<Action>, any>({
-            fetchMyChannelsAndMembers,
-            fetchAllMyTeamsChannelsAndChannelMembers,
-            getMyTeamUnreads,
-            viewChannel,
-            markChannelAsReadOnFocus,
-            getTeamByName,
-            addUserToTeam,
-            setPreviousTeamId,
-            selectTeam,
-            loadStatusesForChannelAndSidebar,
-            getAllGroupsAssociatedToChannelsInTeam,
-            getAllGroupsAssociatedToTeam,
-            getGroupsByUserIdPaginated,
-            getGroups,
-        }, dispatch),
-    };
-}
+const mapDispatchToProps = {
```

Comment:
Neat! writing the `bindActionCreators` boilerplate has long been my least favorite part of how we make a smart component.

Reply:
Tune in for Web Guild on Monday since I wanted to discuss doing this everywhere. I've been hesitant about changing this everywhere, but it helps make our lives so much easier while also helping further isolate the component code from Redux logic